### ISSUE TITLE
[a11y]: labels and alt text are missing 

### ISSUE THREAD

**Issue by CBID2** [codedex-io/css-101#51]
### Description 
In the [HTML file for the lines & blocks lesson](https://github.com/codedex-io/css-101/blob/main/4-layout/19-lines-and-blocks/index.html), there are a couple of accessibility issues that I found:

1. The image is missing alt text. This makes it difficult for screen reader users to understand what the image is describing and how it fits the exercise 
2. The inputs are missing associated labels, which causes screen reader users to [not understand its purpose](https://www.w3.org/WAI/tutorials/forms/labels/)
